normandy/conftest.py

```diff
@@ -4,8 +4,10 @@
 
 import pytest
 import requests_mock
+from graphene.test import Client
```

Comment:
Maybe `import Client as GrapheneClient`? It's not obvious what this is a client *for* where it's actually used.